src/coreComponents/managers/ObjectManagerBase.hpp

```diff
@@ -610,22 +610,21 @@ class ObjectManagerBase : public dataRepository::Group
    * @return A const reference to a view to const data.
    */
   template< typename MESH_DATA_TRAIT >
-  auto const & getExtrinsicData() const
+  GEOSX_DECLTYPE_AUTO_RETURN getExtrinsicData() const
   {
-    return this->getWrapper< typename MESH_DATA_TRAIT::type >( MESH_DATA_TRAIT::key )->referenceAsView();
+    return this->getWrapper< typename MESH_DATA_TRAIT::type >( MESH_DATA_TRAIT::key )->reference();
```

Comment:
Why not `referenceAsView`?

Reply:
It's the same thing here since the method and hence the `Wrapper` returned by `getWrapper` is `const`.